command/alias.go

```diff
@@ -16,21 +16,36 @@ func init() {
 	aliasCmd.AddCommand(aliasSetCmd)
 	aliasCmd.AddCommand(aliasListCmd)
 	aliasCmd.AddCommand(aliasDeleteCmd)
+
+	aliasSetCmd.Flags().BoolP("shell", "s", false, "Declare an alias to be passed through a shell interpreter")
 }
 
 var aliasCmd = &cobra.Command{
 	Use:   "alias",
-	Short: "Create shortcuts for gh commands",
+	Short: "Create command shortcuts",
+	Long: heredoc.Doc(`
+	Aliases can be used to make shortcuts for gh commands or to compose multiple commands.
+
+	Run "gh help alias set" to learn more.
+	`),
 }
 
 var aliasSetCmd = &cobra.Command{
 	Use:   "set <alias> <expansion>",
 	Short: "Create a shortcut for a gh command",
-	Long: `Declare a word as a command alias that will expand to the specified command.
+	Long: heredoc.Doc(`
+	Declare a word as a command alias that will expand to the specified command(s).
 
-The expansion may specify additional arguments and flags. If the expansion
-includes positional placeholders such as '$1', '$2', etc., any extra arguments
-that follow the invocation of an alias will be inserted appropriately.`,
+	The expansion may specify additional arguments and flags. If the expansion
+	includes positional placeholders such as '$1', '$2', etc., any extra arguments
+	that follow the invocation of an alias will be inserted appropriately.
+
+	If '--shell' is specified, the alias will be run through a shell interpreter (sh or pwsh). This allows you
+	to compose commands with "|" or redirect output. Note that extra arguments are not passed to
+	shell-interpreted aliases; only placeholders ("$1", "$2" on *nix, "$args" in powershell) are
```

Comment:
“Note that extra arguments are not passed” - I find this to be misleading. Extra arguments are actually passed and available to shell aliases; it's just that they need to be explicitly placed into the expanded expression. Maybe you could change the wording to reflect that? Something in the lines of: 

> Note that any extra arguments following the alias will not be automatically appended to the expanded expression. To have the alias expansion receive arguments, use `$1`, `$2`, etc. for positional arguments or `"$@"` to place all arguments.